lambdas/gro-convert-to-json/src/test/java/uk/gov/di/data/lep/GroConvertToJsonTest.java

```diff
@@ -133,6 +134,14 @@ class GroConvertToJsonTest {
         "</DeceasedBirthDate>" +
         "</DeathRegistration>" +
         "</DeathRegistrationGroup>";
+    private static final String mockS3objectResponseNoRecords =
+        "<DeathRegistrationGroup xmlns=\"http://www.ons.gov.uk/gro/OGDDeathExtractDWP\" xmlns:ns1=\"http://www.govtalk.gov.uk/people/PersonDescriptives\" xmlns:ns2=\"http://www.govtalk.gov.uk/people/AddressAndPersonalDetails\" xmlns:ns3=\"http://www.ons.gov.uk/gro/people/GROAddressDescriptives\" xmlns:ns4=\"http://www.ons.gov.uk/gro/people/GROPersonDescriptives\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://www.ons.gov.uk/gro/OGDDeathExtractDWP .\\OGDDeathExtractDWP-v1-1.xsd\">" +
+        "<RecordCount>0</RecordCount>" +
+        "</DeathRegistrationGroup>";
+    private static final String mockS3objectResponseExpectedRecordsButFoundNone =
+        "<DeathRegistrationGroup xmlns=\"http://www.ons.gov.uk/gro/OGDDeathExtractDWP\" xmlns:ns1=\"http://www.govtalk.gov.uk/people/PersonDescriptives\" xmlns:ns2=\"http://www.govtalk.gov.uk/people/AddressAndPersonalDetails\" xmlns:ns3=\"http://www.ons.gov.uk/gro/people/GROAddressDescriptives\" xmlns:ns4=\"http://www.ons.gov.uk/gro/people/GROPersonDescriptives\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://www.ons.gov.uk/gro/OGDDeathExtractDWP .\\OGDDeathExtractDWP-v1-1.xsd\">" +
+        "<RecordCount>6</RecordCount>" +
```

Comment:
The other example xml's don't seem to have a record count - is there a reason for this?

Reply:
This is a mistake from previous testing as we did not check the values were equivalent and so did not need the RecordCount for the tests before. These will be added in when the checks for record count are implemented (next ticket).